### PR TITLE
fix: Fix issues on control & perception and improvement on handling control message.

### PR DESCRIPTION
**Pull request by Kim-mins**
## Description

- Fix oscillation issue that makes oscillation while driving.
- Fix perception issue that makes the pointcloud range to narrow.

Here's the video for better description:
- [video with issue](https://drive.google.com/file/d/1oH7uPnrIUfpewWLp3dpS2NCvU15TyITo/view?usp=sharing)
   - The vehicle oscillates to left and right.
   - The range of the pointcloud is too narrow, which could make obstacle detection fail.
- [video after this PR](https://drive.google.com/file/d/1k6-Cmgg43zc5pkPwq6HFd12J7NGc6YJJ/view?usp=sharing)
  - The vehicle does not oscillate anymore.
  - The range of the pointcloud is wide enough, that could make obstacle detection as intended.

## Tests performed
I respected the environment setting and followed testing steps written [here](https://github.com/autowarefoundation/autoware.universe/pull/6859#issue-2255769600).

## Effects on system behavior

This resolves issue on oscillation and pointcloud range during the co-simulation using Carla, as the video above.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: simulator/carla_autoware/README.md [hunk 1]
```diff
@@ -127,7 +127,6 @@ The `carla_ros.py` sets up the CARLA world:
 ## Tips
 
 - Misalignment might occurs during initialization, pressing `init by gnss` button should fix it.
-- Changing the `fixed_delta_seconds` can increase the simulation tick (default 0.05 s), some sensors params in `objects.json` need to be adjusted when it is changed (example: LIDAR rotation frequency have to match the FPS).
```
Comment (Kim-mins) [inline simulator/carla_autoware/README.md]: After [this commit](https://github.com/mraditya01/autoware.universe/commit/84f286d845bb23769e55dfc0cb564c844162df2c), it seems we should fix the `rotation_frequency`.

Comment (maxime-clem) [reply simulator/carla_autoware/README.md]: I think this tip should not be removed from the README.

**Review by maxime-clem — APPROVED** [review]
Thank you very much for your contribution. I tested your changes and the control is now perfect.
I have concerns about the performance impact of increasing the channels of the lidar, but we can add some info in the `tips` about performance in another PR.

File: simulator/carla_autoware/README.md (hunk 1 above)
Comment (Kim-mins) [reply simulator/carla_autoware/README.md]: Thank you for the comment!
I agree to your point. I won't remove it.

**Pull request merged**